dashboard/app/controllers/level_starter_assets_controller.rb

```diff
@@ -1,8 +1,11 @@
 class LevelStarterAssetsController < ApplicationController
+  authorize_resource class: false, except: [:show, :file]
+  before_action :require_levelbuilder_mode, except: [:show, :file]
   before_action :set_level
 
   S3_BUCKET = 'cdo-v3-assets'.freeze
   S3_PREFIX = 'starter_assets/'.freeze
+  VALID_FILE_EXTENSIONS = %w(.jpg .jpeg .gif .png)
```

Comment:
I'd suggest including `.mp3`, assuming that'll essentially give us the ability to upload initial sounds "for free". if that doesn't work, then let's at least make sure something sane happens in the ui when a levelbuilder tries to upload a sound file in start mode. again this can be a separate PR if needed

Reply:
good call. we wanted to scope this feature to only image assets, but uploading/rendering an .mp3 works just fine, so we might as well add that to the list of valid extensions!